modules/serverless-invoking-quarkus-functions.adoc

```diff
@@ -0,0 +1,157 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-quarkus-functions.adoc
+
+[id="serverless-invoking-quarkus-functions_{context}"]
+= About invoking Quarkus functions
+
+You can create a Quarkus project that responds to CloudEvents, or one that responds to simple HTTP requests. CloudEvents in Knative are transported over HTTP as a POST request, so either function type can listen and respond to incoming HTTP events.
```

Comment:
should it be s/incoming HTTP events/incoming HTTP requests ? (not sure about this one)

Reply:
Should be requests I think, good catch